uint32 src_vehicle
uint32 platoon_id
float64 ipd
float64 pd